docs/docs/misc/glossary/call_types.md

```diff
@@ -0,0 +1,177 @@
+
+#### `eth_call`
+
+This method is the largest culprit of confusion around calls, but unfortunately requires understanding of all previous concepts in order to be explained. Its name is also quite unhelpful.
+
+What `eth_call` does is simulate a transaction (a call to a contract) given the current blockchain state. The behavior will be the exact same as `eth_sendTransaction`, except:
+
+- no actual transaction will be created
+- while gas _will_ be measured, there'll be no transaction fees of any kind
+- no signature is required: the `from` address is passed directly, and can be set to any value (even if the private key is unknown, or if they are contract addresses!)
+- the return value of the called contract is available
+
+`eth_call` is typically used for one of the following:
+
+- query blockchain data, e.g. read token balances
+- preview the state changes produced by a transaction, e.g. the transaction cost, token balance changes, etc
+
+Because some libraries ([such as ethers](https://docs.ethers.org/v5/getting-started/#getting-started--reading)) automatically use `eth_call` for `view` functions (which when called via Solidity result in the `STATICCALL` opcode), these concepts can be hard to tell apart. The following bears repeating: **an `eth_call`'s call context is the same as `eth_sendTransaction`, and it is a `CALL` context, not `STATICCALL`.**
+
+## Aztec Call Types
+
+Large parts of the Aztec Network's design are still not finalized, and the nitty-gritty of contract calls is no exception. This section won't therefore contain a thorough review of these, but rather list some of the main ways contracts can currently be interacted with, with analogies to Ethereum call types when applicable.
+
+While Ethereum contracts are defined by bytecode that runs on the EVM, Aztec contracts have multiple modes of execution depending on the function that is invoked.
+
+### Private Execution
+
+Contract functions marked with `#[aztec(private)]` can only be called privately, and as such 'run' in the user's device. Since they're circuits, their 'execution' is actually the generation of a zk-SNARK proof that'll later be sent to the sequencer for verification.
+
+#### Private Calls
+
+Private functions from other contracts can be called either regularly or statically by using the `.call()` and `.static_call` functions. They will also be 'executed' (i.e. proved) in the user's device, and `static_call` will fail if any state changes are attempted (like the EVM's `STATICCALL`). 
+
+#include_code private_call /noir-projects/noir-contracts/contracts/lending_contract/src/main.nr rust
+
+Unlike the EVM however, private execution doesn't revert in the traditional way: in case of error (e.g. a failed assertion, a state changing operation in a static context, etc.) the proof generation simply fails and no transaction request is generated, spending no network gas or user funds.
+
+#### Public Calls
```

Comment:
Should the static call be mentioned under public as well, in private you mention it.